lib/matplotlib/offsetbox.py

```diff
@@ -596,6 +596,15 @@ def __init__(self, width, height, xdescent=0.,
 
         self.dpi_transform = mtransforms.Affine2D()
 
+    @property
+    def clip_children(self):
+        return self._clip_children
+
+    @clip_children.setter
+    def clip_children(self, val):
+        self._clip_children = bool(val)
+        self.stale = True
```

Comment:
Yes, but there are only a few checks anywhere else.  This problem will either be solved if we go with a managed attributes meta-class (i.e. traitlets) or slowly as it annoys people.